docs/src/pages/demos/tables/ReactVirtualizedTable.tsx

```diff
@@ -0,0 +1,239 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { withStyles, WithStyles, Theme, createStyles } from '@material-ui/core/styles';
+import TableCell from '@material-ui/core/TableCell';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import Paper from '@material-ui/core/Paper';
+import { AutoSizer, Column, SortDirection, Table } from 'react-virtualized';
```

Comment:
I am getting a `Could not find a declaration file for module 'react-virtualized'` error on this module.

Reply:
[@types/react-virtualized](https://www.npmjs.com/package/@types/react-virtualized) should be added